src/tools/pontoon/renaming_a_project.md

```diff
@@ -0,0 +1,12 @@
+# Renaming a project
+
+You need to think carefully on the name when setting up a new project. Though rare, sometimes a project needs to be renamed because the product name has changed or for some other reasons. To rename it, it’s quite simple:
```

Comment:
A couple of notes: 
* It misses the "why" you need to think carefully (because the `slug` is generated automatically from the name, unless you manually edit it)
* Colon followed by a new paragraph doesn't make a lot of sense, also "it's simple" should be avoid where possible ;-) `To rename it, got to the …`.